plugins/blip/basics/logic/constants.js

```diff
@@ -6,5 +6,13 @@ module.exports = {
   NO_CGM: 'noCGM',
   NO_SITE_CHANGE: 'noSiteChange',
   NOT_ENOUGH_CGM: 'notEnoughCGM',
-  SITE_CHANGE: 'siteChange'
+  SITE_CHANGE: 'siteChange',
+  SITE_CHANGE_RESERVOIR: 'reservoirChange',
+  SITE_CHANGE_TUBING: 'tubingPrime',
+  SITE_CHANGE_CANNULA: 'cannulaPrime',
+  TYPE_UNDECLARED: 'undeclared',
```

Comment:
`SITE_CHANGE_SOURCE_UNDECLARED` perhaps? Not sure I've fully groked the usage yet without diving into the code in more detail.